templates/profile/show.html.twig

```diff
@@ -0,0 +1,70 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheets %}
+	{{ parent() }}
+	{{ encore_entry_link_tags('profile') }}
+{% endblock %}
+
+{% block title %}
+	{{ parent() }}
+	-
+	{{ registeredUser.username }}
+{% endblock %}
+
+{% block body %}
+	<main class="p-lg-5 p-3">
+		<div class="profile bg-dark p-lg-5 p-4">
+			<div class="profile-header d-flex justify-content-between">
+				<div class="d-flex flex-direction-row align-items-center">
+					<img class="rounded-circle me-4" src="https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png" alt="avatar-user">
+					<div class="profile-titles">
+						<h1 class="h1 text-warning m-0">{{ registeredUser.firstname }}
+							{{ registeredUser.lastname }}</h1>
+						<h2 class="h3 m-0">Développeuse PHP</h2>
+					</div>
+				</div>
+				<i class="far fa-heart"></i>
+			</div>
+			<div class="profile-body mt-5">
+				<div class="mb-5">
+					<div class="d-flex flex-direction-column align-items-center mb-4">
+						<i class="fas fa-briefcase me-4"></i>
+						<p class="m-0 p-0">Orléans, Loiret</p>
+					</div>
+					<div class="d-flex flex-direction-column align-items-center mb-4">
+						<i class="fas fa-home me-4"></i>
+						<p class="m-0 p-0">Tours, Indre-et-Loire</p>
```

Comment:
isn't this specified dynamically thanks to the "address" property in "registeredUser"?